### PR TITLE
Add source url and add to scrapers

### PR DESCRIPTION
**Pull request by jenlu33**
Asana ticket: [https://app.asana.com/0/1203289004376659/1204375229874464/f](https://app.asana.com/0/1203289004376659/1204375229874464/f)

I added a separate source url to StandardsImports and added it to the scrapers.

### CODE REVIEW

File: app/jobs/scraper/oregon_job.rb [hunk 1]
```diff
@@ -38,7 +38,8 @@ def perform
             organization: organization
           ).first_or_initialize(
             notes: "From Scraper::OregonJob",
-            public_document: true
+            public_document: true,
+            source_url: apprenticeship_url
```
Comment (littleforest) [inline app/jobs/scraper/oregon_job.rb:42]: I'm wondering if this should be `apprenticeship_url + "apprenticeship-opportunities.aspx"` ?

**Review by littleforest — COMMENTED** [review]
Looks good! Just one question.

File: app/jobs/scraper/oregon_job.rb (hunk 1 above)
Comment (jenlu33) [reply app/jobs/scraper/oregon_job.rb:42]: That makes sense. I'll add that in

**Review by littleforest — APPROVED** [review]
🚢

**Pull request merged**